src/lib/core/fiber.c

```diff
@@ -1879,6 +1985,11 @@ fiber_init(int (*invoke)(fiber_func f, va_list ap))
 	main_cord.loop = ev_default_loop(EVFLAG_AUTO | EVFLAG_ALLOCFD);
 	cord_create(&main_cord, "main");
 	fiber_signal_init();
+#ifndef NDEBUG
+	fiber_debug_enabled = true;
+#else
+	fiber_debug_enabled = false;
+#endif
```

Comment:
Nit: this would probably be done where this variable is defined.

Reply:
Changed to

```c
#ifndef NDEBUG
bool fiber_debug_enabled = true;
bool fiber_abort_on_gc_leak = true;
#else
bool fiber_debug_enabled = false;
bool fiber_abort_on_gc_leak = false;
#endif
```